windows/src/global/delphi/general/Keyman.System.Settings.pas

```diff
@@ -0,0 +1,583 @@
+    ),
+
+    //
+    // development.urls
+    //
+
+    // Upload_Settings.API_Protocol
+    (
+      ID: 'development.urls.api_keyman_com_protocol';
+      Name: 'Debug_APIProtocol';
+      RootKey: HKCU;
+      Key: SRegKey_KeymanDebug_CU;
+      Description: 'Protocol for api.keyman.com; use this when debugging '+
+                   'Keyman with a local api.keyman.com instance';
+      DefaultStr: 'https'
+    ),
+
+    // Upload_Settings.API_Server
+    (
+      ID: 'development.urls.api_keyman_com_host';
+      Name: 'Debug_APIServer';
+      RootKey: HKCU;
+      Key: SRegKey_KeymanDebug_CU;
+      Description: 'Host for api.keyman.com; use this when debugging '+
+                   'Keyman with a local api.keyman.com instance';
+      DefaultStr: 'api.keyman.com'
+    ),
+
+    // Upload_Settings.KeymanCom_Protocol_Server
+    (
+      ID: 'development.urls.keyman_com';
+      Name: 'Debug_KeymanCom';
+      RootKey: HKCU;
+      Key: SRegKey_KeymanDebug_CU;
+      Description: 'URL to keyman.com; use this when debugging Keyman with a '+
+                   'local keyman.com instance';
+      DefaultStr: 'https://keyman.com'
+    ),
+
+    // TIKE:RedistFiles.GetHelpURL
+    (
+      ID: 'development.urls.developer.help';
+      Name: 'Debug_HelpURL';
+      RootKey: HKCU;
+      Key: SRegKey_KeymanDebug_CU;
+      Description: 'Base URL for Keyman Developer documentation';
+      DefaultStr: 'https://keyman.com/go/developer/'+SKeymanVersion+'/docs'
+    ),
+
+    //
+    // engine.compatibility
+    //
+
+    // kmcomapi:keymancontrol.TKeymanControl.LoadKeyman32.GetKeymanInstallPath
+    // keyman32_int.GetKeymanInstallPath
+    (
+      ID: 'engine.compatibility.text_services_framework';
+      Name: 'deep tsf integration';
+      RootKey: HKLM;
+      Key: SRegKey_KeymanEngine_LM;
+      Description: 'If present, 0=disabled; 1=enabled. Learn more at '+
+                   'https://help.keyman.com/kb/94';
+      DefaultInt: 1;
+      ValueType: kstInteger
+    ),
+
+    // keyman:TfrmKeyman7Main.RegisterHotkeys,
+    // keyman:TfrmKeyman7Main.UnregisterHotkeys
+    (
+      ID: 'engine.compatibility.old_hotkey_registration';
+      Name: SRegValue_Flag_UseRegisterHotkey;
+      RootKey: HKCU;
+      Key: SRegKey_KeymanEngineDebug_CU;
+      Description: 'Set to 1 for old RegisterHotkey pathway';
+      ValueType: kstInteger
+    ),
+
+    // keyman32:keyman32.Initialise_Flag_ShouldSerializeInput
+    (
+      ID: 'engine.compatibility.serialize_input';
+      Name: SRegValue_Flag_ShouldSerializeInput;
+      RootKey: HKCU;
+      Key: SRegKey_KeymanEngineDebug_CU;
+      Description: 'Set to 0 to avoid serializing input. See '+
+                   'https://blog.keyman.com/2018/10/the-keyman-keyboard-input-'+
+                   'pipeline/ for details';
+      DefaultInt: 1;
+      ValueType: kstInteger
+    ),
+
+    //
+    // engine.diagnostics
+    //
+
+    // keyman32:k32_globals.Globals::LoadDebugSettings
```

Comment:
You'll note that this is the only setting which is also available in the Keyman Options.